src/recaptcha-wrapper.js

```diff
@@ -35,6 +35,28 @@ export function createRecaptcha () {
       this.assertLoaded()
       this.wait().then(() => window.grecaptcha.execute(widgetId))
     },
+	
+	loadRecaptcha () {
+		let recaptchaDetected = false;
+		let recaptchaSrc = 'https://www.google.com/recaptcha/api.js?onload=vueRecaptchaApiLoaded&render=explicit';
+		let documentScripts = document.getElementsByTagName('script');
+		if (documentScripts.length) {
+			for (let documentScript of documentScripts) {
+				if (documentScript.src === recaptchaSrc) {
+					recaptchaDetected = true;
+					return;
```

Comment:
Since here is a early return, we can remove `recaptchaDetected` and the condition below.